app/middlewares/auth.js

```diff
@@ -0,0 +1,29 @@
+const errors = require('../errors'),
```

Comment:
A very similar middleware has been added in the list PR, maybe you should make this branch to depend on that one, to avoid double work.